src/randoop/main/GenInputsAbstract.java

```diff
@@ -201,17 +167,16 @@ public GenInputsAbstract(String command, String pitch,
   /** Do not generate tests with more than this many statements */
   @Option("Do not generate tests with more than <int> statements")
   public static int maxsize = 100;
-  
-
 
   /**
    * Use null with the given frequency as an argument to method calls.
    * 
    * For example, a null ratio of 0.05 directs Randoop to use
    * <code>null</code> as an input 5 percent of the time when a
-   * non-<code>null</code> value of the appropriate type is available.  If
-   * no non-<code>null</code> value is available, Randoop will try
-   * <code>null</code> regardless of the value of this parameter.
+   * non-<code>null</code> value of the appropriate type is available.
+   * 
```

Comment:
Note that line breaks introduced in the Javadoc code comments will not appear in either the HTML API documentation or in the manual unless you add an explicit `<p>`.  This may be intended only for the code comments, but it's a bit inconsistent and confusing to have line breaks that are considered important enough for the code comments but not for the formatted documentation.   I would be inclined to remove them for consistency.  (I realize that they predate your changes.)